client/app/views/steps/password.coffee

```diff
@@ -103,9 +103,13 @@ module.exports = class PasswordView extends StepView
             onboardedSteps: ['welcome', 'agreement', 'password']
         }
 
-    renderError: (error) ->
+    renderError: (error, useTranslation = true) ->
```

Comment:
What do you think about the idea of letting the server return a translation key instead of a translated message ? It should be less human-friendly but maybe more like what an API could return.